.github/workflows/mirror-docker-image.yml

```diff
@@ -0,0 +1,37 @@
+name: mirror-docker-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        type: string
+        description: |
+          oras-compatible image ref to mirror (ex. 'docker.io/library/busybox')
+      version:
+        type: string
+        description: |
+          Version of the source image to mirror (ex. '@<digest>', ':<tag>')
+      output-image:
+        description: |
+          Image path under 'wasmcloud/mirrors' to use (ex. 'a' -> 'wasmcloud/mirrors/a:<tag>', 'tools/b' -> 'wasmcloud/mirrors/tools/b:<tag>')
+        optional: true
+
+permissions: {}
+
+jobs:
+  mirror-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - uses: oras-project/setup-oras@ca28077386065e263c03428f4ae0c09024817c93
+      - name: Docker login
+        shell: bash
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | oras login ghcr.io --username ${{ github.repository_owner }} --password-stdin
+      - name: Copy image
+        shell: bash
+        run: |
+          oras cp \
+          ${{ inputs.image }}${{ inputs.version }} \
```

Comment:
should this be:
```suggestion
          ${{ inputs.image }}:${{ inputs.version }} \
```

Reply:
Oh I see, the expectation is for folks to remember to add `:`

I think it would be worth adding a check to validate that the `input.version` starts with `:` or `@` and error the action out if it doesn't to make it obvious when you might've have accidentally it out.

Reply:
Yeah that's pretty reasonable, I can add a step to validate that they're using the right prefix there!

- [ ] Add check of prefix for version